### PR TITLE
Multi-ouput Gaussian Process Coregion models with Hadamard product

### PR DESCRIPTION
**Pull request by danhphan**
This PR adds an example of how to implement Intrinsic Coregionalization Model (ICM) and Linear Coregionalization Model (LCM) using a Hamadard product between Coregion kernel and input kernels. 

We model the average spin rates of several pitchers on different game dates from a baseball dataset.

Cc @bwengals @fonnesbeck 

Let me know if it needs to update anythings.

Thank you.

### CODE REVIEW

**Comment by danhphan** [comment]
<p>Hi yes, thanks Chris, I will address these comments this weekend! </p>

**Comment by danhphan** [comment]
Thank you Chris, I have updated the notebook according to your suggestions. Let me know if it needs any further changes.

Also will need one more reviewer for this notebook. Thanks!

**Comment by danhphan** [comment]
<p>Hi Bill, I would like to keep the authors like that. Since several parts of this notebook are from your previous codes of ICM on PyMC3. Besides, it makes sense to include the mentors, as they also involve a lot in the ideas and suggestions of the notebook's content. Thank you.</p>

---

 <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/454/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by danhphan** [comment]
<p>Hi, thank you Bill, I have updated it :)</p>

File: myst_nbs/gaussian_processes/MOGP-Coregion-Hadamard.myst.md [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+
+## Authors
+* Authored by [Danh Phan](https://github.com/danhphan), [Bill Engels](https://github.com/bwengals), [Chris Fonnesbeck](https://github.com/fonnesbeck) in November, 2022 ([pymc-examples#454](https://github.com/pymc-devs/pymc-examples/pull/454))
+
++++
+
+## References
+:::{bibliography}
+:filter: docname in docnames
+:::
```
Comment (OriolAbril) [inline myst_nbs/gaussian_processes/MOGP-Coregion-Hadamard.myst.md:347]: There are no references cited in the notebook, so this part should be skipped.

File: myst_nbs/gaussian_processes/MOGP-Coregion-Hadamard.myst.md [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+
```
Comment (OriolAbril) [inline myst_nbs/gaussian_processes/MOGP-Coregion-Hadamard.myst.md:348]: Right before this there should be the watermark title: https://docs.pymc.io/en/latest/contributing/jupyter_style.html#watermark

**Review by OriolAbril — COMMENTED** [review]
left two very minor comments

File: myst_nbs/gaussian_processes/MOGP-Coregion-Hadamard.myst.md (hunk 1 above)
Comment (fonnesbeck) [reply myst_nbs/gaussian_processes/MOGP-Coregion-Hadamard.myst.md:347]: Might be worth adding a multi-output GP reference, actually, for those that want more background (even if its just the relevant page of the GPyTorch docs).

Comment (danhphan) [reply myst_nbs/gaussian_processes/MOGP-Coregion-Hadamard.myst.md:347]: Hi yes, I have added a reference, also mentioned further related materials at the end of the slides :)

**Comment by danhphan** [comment]
Thanks @OriolAbril. The notebook has been updated with reference and watermark :)

**Comment by danhphan** [comment]
Hi @OriolAbril @fonnesbeck If no further changes, this PR should be good to be merged. Thank you!

**Review by fonnesbeck — APPROVED** [review]
LGTM

**Comment by danhphan** [comment]
Hi, thank you @fonnesbeck !

**Pull request merged**